### ISSUE TITLE
Save category data in local storage. There is no account to save this per user right now. Hopefully this works. Maybe I could allow them to download the category data as a csv so they can load it again next time.

### ISSUE THREAD

**Issue by wsurles** [wsurles/finance_tracker#13]

**Comment by greentheo** [comment]
What about a redis database?  It's a key value store and readily accessible in R.  I guess it needs to run on a server though... we could put one up on my local server (the internet isn't 100% reliable though).

Check out rredis...

**Comment by greentheo** [comment]
this site says that they have a free one... that would probably work for this.  Hmm.. maybe I'll switch my one app over to it.  https://redislabs.com/

**Comment by wsurles** [comment]
I have not used this before but will check it out. I'll need to handle
account logins somehow eventually.

**Comment by greentheo** [comment]
yeah... my friend kent has been doing some work on login stuff for a data collection and reporting app we've been working on.  

We could most likely repurpose it for this app as well.

**Comment by wsurles** [comment]
That would be sweet. Its critical to many data apps really. Especially
anything in finance. Would be good to know how to do that.
